### ISSUE TITLE
RT95 cooling possibilities

### ISSUE THREAD

**Issue by EXCV31** [CircuitChaos/OpenMicron#2]
Hi :)

I saw your video on youtube with CRT dissasembly. I get my radio for present, but i noticed that RT95 gets pretty warm when you transmit 25W. Of course i don't push PTT for 15 minutes, but in normal QSO. Did you find any possibilities to upgrade cooling system, maybe with some fan on external side of radiator?

**Comment by CircuitChaos** [comment]
Hey :) I didn't upgrade the cooling system, but I'm using AT-778UV monthly in a contest on 2m, during which I transmit almost constantly (with 25 W) and I didn't notice it getting hot. It gets warm, but not too warm, and it doesn't burn when I touch it. Maybe your SWR is too high?

**Comment by EXCV31** [comment]
Thank you for response. I checked everything, antenna SWR is 1.3, the problem turned out to be the sun, through which the black metal housing heats up so much. I did tests in the evening at 20*C and the radio gets barely warm. I'm going to think now about some piece of metal sheet painted white, so that it reflects the sun's waves and doesn't heat the radio so much. Thanks for the direction in solving the problem, 73!
